test_app/models.py

```diff
@@ -17,6 +17,7 @@ class ExamplesChild(models.Model):
         on_delete=models.CASCADE,
         related_name="children",
         related_query_name="children",
+        verbose_name="parent alias",
```

Comment:
This now changes every test to include a related field with a verbose name, but we lose the test without the verbose name. Is there a way to do this so that we test both cases? Or do we think it's not needed/useful?

Reply:
don't think we need an extra test. by setting `verbose_name`, the serializer runs the following extra lines. so it is additive and not xor. 

```
if model_field.verbose_name and needs_label(model_field, field_name):
    kwargs["label"] = capfirst(model_field.verbose_name)
```